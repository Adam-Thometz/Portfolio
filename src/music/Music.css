.Music {
  width: 80%;
  margin: 0 auto;
  text-align: center;
}

.Music-songs {
  display: flex;
  justify-content: space-evenly;
}