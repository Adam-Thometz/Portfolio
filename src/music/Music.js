import React from "react";

import "./Music.css";

const Music = ({ style }) => {
  return (
    <section className="Music" style={style}>
      <h2>Music</h2>
      <div className="Music-songs">
        <iframe
          title="Night Cycle"
          width="100%"
          height="200"
          scrolling="no"
          frameborder="no"
          allow="autoplay"
          src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/826679479&color=%23ff5500&auto_play=false&hide_related=false&show_comments=true&show_user=true&show_reposts=false&show_teaser=true&visual=true"
        ></iframe>
        <iframe
          title="Prime Soup"
          width="100%"
          height="200"
          scrolling="no"
          frameborder="no"
          allow="autoplay"
          src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/750036325&color=%23ff5500&auto_play=false&hide_related=false&show_comments=true&show_user=true&show_reposts=false&show_teaser=true&visual=true"
        ></iframe>
      </div>
    </section>
  );
};

export default Music;
