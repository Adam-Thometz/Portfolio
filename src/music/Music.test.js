import React from "react";
import { render } from "@testing-library/react";
import Music from "./Music";

describe("Music component", () => {
  it("renders without crashing", () => {
    render(<Music />);
  });

  it("matches the snapshot", () => {
    const { asFragment } = render(<Music />);
    expect(asFragment()).toMatchSnapshot();
  });
});
