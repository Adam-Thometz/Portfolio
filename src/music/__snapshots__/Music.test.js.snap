// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Music component matches the snapshot 1`] = `
<DocumentFragment>
  <section
    class="Music"
  >
    <h2>
      Music
    </h2>
    <iframe
      allow="autoplay"
      frameborder="no"
      height="300"
      scrolling="no"
      src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/826679479&color=%23ff5500&auto_play=false&hide_related=false&show_comments=true&show_user=true&show_reposts=false&show_teaser=true&visual=true"
      title="Night Cycle"
      width="100%"
    />
    <iframe
      allow="autoplay"
      frameborder="no"
      height="300"
      scrolling="no"
      src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/750036325&color=%23ff5500&auto_play=false&hide_related=false&show_comments=true&show_user=true&show_reposts=false&show_teaser=true&visual=true"
      title="Prime Soup"
      width="100%"
    />
  </section>
</DocumentFragment>
`;
